### ISSUE TITLE
Configuration dialog of the mac os x client is not accessible

### ISSUE THREAD

**Issue by guillaume-ggo** [owncloud/client#736]
With the built-in screen reader of Apple Mac Os X, I cannot to login to owncloud because the configuration dialog is not accessible.
I can't navigate through the interface and so I cannot enter my username and password.
Probably the problem is the accessibility support of QT?
